modules/manually-create-identity-access-management.adoc

```diff
@@ -23,8 +23,7 @@ installation in environments where the cloud identity and access management
 administrator-level credential secret in the cluster `kube-system` namespace.
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.